### DISCUSSION TITLE
Do you want to collaborate to improve Score Compose?

### DISCUSSION THREAD

**Discussion by gguizza** [score-spec/spec#57] [General]
Hey, long time no see! 
The team has been working behind the scenes to understand the areas of improvement of Score as it is right now. 

We want to improve the user experience of [score-compose](https://github.com/score-spec/score-compose). We'd love to learn more about your setup and regular workflow with Docker Compose or score-compose and any pain points you've faced.

Let's collaborate and shape the future of Score together!

**Comment by taras** [comment]
I shared this a few times in different places, so I'll leave the same feedback here. Using `score-compose` requires invoking docker-compose with specific arguments. It would be more convenient if someone could just run `score-compose run`, and it would automatically invoke the docker-compose binary with correct arguments.
